docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   server:
-    image: doorbot-challenge
+    image: thann/doorbot
```

Comment:
You don't need a preexisting tag name, you're supposed to build your own image with that name ^.^